api/src/main/java/io/opencensus/trace/unsafe/ContextManagerImpl.java

```diff
@@ -0,0 +1,39 @@
+package io.opencensus.trace.unsafe;
+
+import io.grpc.Context;
+import io.opencensus.trace.ContextManager;
+import io.opencensus.trace.Ctx;
+import io.opencensus.trace.Span;
+import javax.annotation.Nullable;
+
+/**
+ * Default {@code ContextManager} implementation using {@see io.grpc.Context}
+ */
+public class ContextManagerImpl implements ContextManager {
```

Comment:
Default implementation simply redirects all calls to `ContextUtils` (original implementation).